src/edu/washington/escience/myria/api/encoding/DatasetEncoding.java

```diff
@@ -23,4 +25,5 @@
   public Boolean importFromDatabase;
   public List<List<IndexRef>> indexes;
   public Boolean overwrite;
+  public PartitionFunction howPartitioned = new RoundRobinPartitionFunction(null);
```

Comment:
I'm confused -- isn't this the partition function for ingest? `howPartitioned` does not make sense as a field name.